internal/drivers/manager.go

```diff
@@ -521,6 +521,13 @@ func (m *Manager) setupInstance(ctx context.Context, pool *poolEntry, inuse bool
 			Errorln("manager: failed to generate certificates")
 		return nil, err
 	}
+	maxRetries := 3
```

Comment:
I feel this should be set at the driver layer instead of here -  mainly because each driver might have different values or retry logic it wants to use

Reply:
done